lib/drivers/tandemDriver.js

```diff
@@ -979,6 +994,79 @@ module.exports = function (config) {
     }, INTERVAL_FREQ); // if we spin too quickly on this, packets don't get sent when window doesn't have focus
   };
 
+  var multiLogRequester = function (start, end, progress, callback) {
+    if (__DEBUG__) {
+      debug('multiLogRequester', start, end);
+      var start_exec = Date.now();
```

Comment:
So, what is our standard for case, snake_case, camelCase? It looks like the rest of the code uses camelCase. We should be consistent.